### PR TITLE
Anshu-Assignment-23-150min+45min-Done

### PR DESCRIPTION
**Pull request by AnshuKJun20**
image
image

### CODE REVIEW

File: src/anshuKJun20/IntersectionOfArray.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+		int count = 0;
+		for (int outerindex = 0; outerindex < arr2.length; outerindex++) {
+			boolean flag = false;
+			for (int innerindex = 0; innerindex < arr1.length; innerindex++) {
+				if (arr2[outerindex] == arr1[innerindex]) {
+					flag = true;
+					break;
+				}
+			}
+			if (flag == true) {
```
Comment (PranitaPJun20) [inline src/anshuKJun20/IntersectionOfArray.java]: 1.Initially you set flag =false.In first if statement you are changing flag value to 'true' and again you are checking for 
if (flag == true) condition.This not correct.
2.Actually in this code you dont need to set flag value.Without setting also you can get desired output.
3.Line number 19 and 20 merge in first If statement and delete second if statement.

**Review by PranitaPJun20 — COMMENTED** [review]
Please work on given comments.

**Review by PranitaPJun20 — APPROVED** [review]
Corrected as per comments.

File: src/anshuKJun20/IntersectionOfArray.java (hunk 1 above)
Comment (AnshuKJun20) [reply src/anshuKJun20/IntersectionOfArray.java]: You are right, No need of flag here. Changes Done.

File: src/anshuKJun20/IntersectionOfArray.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+			double[] tempArr = new double[arr1.length];
+		}else {
+			double[] tempArr = new double[arr2.length];
+		}
+		int count = 0;
+		for (int outerindex = 0; outerindex < arr2.length; outerindex++) {
+			
+			for (int innerindex = 0; innerindex < arr1.length; innerindex++) {
+				if (arr2[outerindex] == arr1[innerindex]) {
+					tempArr[count] = arr2[outerindex];
```
Comment (KrishnaKTechnocredits) [inline src/anshuKJun20/IntersectionOfArray.java:18]: I think here it should be a compilation error. Please verify.

Comment (AnshuKJun20) [reply src/anshuKJun20/IntersectionOfArray.java:18]: image

I checked it and also debug it. There is no any compilation error.

Comment (AnshuKJun20) [reply src/anshuKJun20/IntersectionOfArray.java:18]: image

**Pull request merged**